redbot/cogs/downloader/downloader.py

```diff
@@ -861,6 +861,31 @@ async def _cog_unpin(self, ctx: commands.Context, *cogs: InstalledCog) -> None:
             message += _("\nThese cogs weren't pinned: ") + humanize_list(not_pinned)
         await self.send_pagified(ctx, message)
 
+    @cog.command(name="listpinned")
+    async def _cog_listpinned(self, ctx: commands.Context):
+        """List currently pinned cogs."""
+        installed = await self.installed_cogs()
+        pinned_list = sorted([cog.name for cog in installed if cog.pinned], key=str.lower)
+        if pinned_list:
+            message = humanize_list(pinned_list)
+        else:
+            message = _("None.")
+        if await ctx.embed_requested():
+            embed = discord.Embed(color=(await ctx.embed_colour()))
+            for page in pagify(message, delims=[", "], page_length=900):
+                name = _("(continued)") if page.startswith(", ") else _("Pinned Cogs:")
+                if page.startswith(", "):
+                    page = page[2:]
+                embed.add_field(name=name, value=page, inline=False)
```

Comment:
This can technically fail if you would have a lot of pinned cogs, but I don't think we would find anyone with over 700 pinned cogs to run into this problem so it's all good 👍